src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -843,9 +843,13 @@ private void recordSelectionForCastVoteRecord(
           selectedCandidate,
           cvr.getFractionalTransferValue());
       if (config.isTabulateByPrecinctEnabled()) {
-        precinctTallyTransfers
-            .get(cvr.getPrecinct())
-            .addTransfer(
+        String precinctName = cvr.getPrecinct();
+        TallyTransfers precinctTallyTransfer = precinctTallyTransfers.get(precinctName);
+        if (precinctTallyTransfer == null) {
+          Logger.severe("Precinct \"%s\" is not among the %d known precincts.",
+                  precinctName, precinctNames.size());
```

Comment:
What is the proper error to throw here? Currently, it'll hit this and get an NPE later.

I don't see existing errors for "tabulation succeeded but writing output failed."